src/AffectedUserFlow/PublishConsent/PublishConsentForm.tsx

```diff
@@ -147,103 +146,91 @@ const PublishConsentForm: FunctionComponent<PublishConsentFormProps> = ({
   }
 
   return (
-    <>
-      <SafeAreaView style={style.topSafeArea} />
-      <SafeAreaView style={style.bottomSafeArea}>
-        <View style={style.outerContainer}>
-          <View style={style.navButtonContainer}>
-            <TouchableOpacity
-              onPress={handleOnPressBack}
-              accessible
-              accessibilityLabel={t("export.code_input_button_back")}
-            >
-              <View style={style.backButtonInnerContainer}>
-                <SvgXml
-                  xml={Icons.ArrowLeft}
-                  fill={Colors.black}
-                  width={Iconography.xSmall}
-                  height={Iconography.xSmall}
-                />
-              </View>
-            </TouchableOpacity>
-
-            <TouchableOpacity
-              onPress={handleOnPressCancel}
-              accessible
-              accessibilityLabel={t("export.code_input_button_cancel")}
-            >
-              <View style={style.cancelButtonInnerContainer}>
-                <SvgXml
-                  xml={Icons.X}
-                  fill={Colors.black}
-                  width={Iconography.xSmall}
-                  height={Iconography.xSmall}
-                />
-              </View>
-            </TouchableOpacity>
+    <View style={style.outerContainer}>
```

Comment:
Should we have a `StatusBar` here since we are using the `useStatusBarEffect` hook?

Reply:
Nope because this screen has a header!